iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/pipe/connector/PipeReceiverStatusHandler.java

```diff
@@ -133,12 +133,14 @@ public void handle(TSStatus status, String exceptionMessage, String recordMessag
 
           LOGGER.warn(
               "User conflict exception: will retry {}. status: {}",
-              (retryMaxMillisWhenConflictOccurs == Long.MAX_VALUE ? "forever" : "for at least ")
-                  + (retryMaxMillisWhenConflictOccurs
-                          + exceptionFirstEncounteredTime.get()
-                          - System.currentTimeMillis())
-                      / 1000.0
-                  + " seconds",
+              retryMaxMillisWhenConflictOccurs == Long.MAX_VALUE
```

Comment:
Personally I think keeping the "()" is clearer...

Reply:
The previous approach would result in appending "xxx seconds" after "forever".